ops/pebble.py

```diff
@@ -2126,7 +2168,9 @@ def autostart_services(self, timeout: float = 30.0, delay: float = 0.1) -> Chang
         return self._services_action('autostart', [], timeout, delay)
 
     def replan_services(self, timeout: float = 30.0, delay: float = 0.1) -> ChangeID:
-        """Replan by (re)starting changed and startup-enabled services and wait for them to start.
+        """Replan by (re)starting changed and startup-enabled services and checks.
```

Comment:
It's a pity I called this `replan_services`, but oh well, at least it's called just `replan` for the `Container` method. Not worth deprecating/aliasing here, I think.

Reply:
Yeah, that was my thinking as well.